insights_client/__init__.py

```diff
@@ -123,6 +123,23 @@ def _main():
     attempt to collect and upload with new, then current, then rpm
     if an egg fails a phase never try it again
     """
+     # flake8 complains because these imports aren't at the top
+    from insights.client import InsightsClient  # noqa E402
+    from insights.client import config  # noqa E402
+
+     # handle client instantation here so that it isn't done multiple times in __init__
+    global client, config, debug
```

Comment:
What if we passed the client into the call to phase?  The benefit would be that we wouldn't need these global vars.  wdyt?